### ISSUE TITLE
Network connection button is hard to find

### ISSUE THREAD

**Issue by tamas646** [TeamAmaze/AmazeFileManager#2348]
### In one sentence: It took me two days to find out how can I add an SFTP connection. :D

Currently it's under the plus button, but I think it's not relevant.
![kép](https://user-images.githubusercontent.com/37243375/110339132-1ab44100-8028-11eb-8621-594a6508dd04.png)

Maybe it would be better to put it in the main menu? (eg. above the shortcuts)
![kép](https://user-images.githubusercontent.com/37243375/110340165-3e2bbb80-8029-11eb-82d4-425adf26bb51.png)

**Comment by EmmanuelMess** [comment]
Against, plus is for adding stuff.

**Comment by EmmanuelMess** [comment]
@VishalNehra @TranceLove ?

**Comment by VishalNehra** [comment]
Yes, for adding new connection, + button makes sense.. populating these things in drawer will spam it. Just think about options such as 'Add SMB connection', 'New SFTP connection' and many more in drawer. However, I do agree that many people face difficulty finding this option.

**Comment by tamas646** [comment]
> Against, plus is for adding stuff.

Yes, but this type of plus button is mainly used to add stuff related to the actual content displayed. Eg. if you're viewing the content of a directory, you expect options like "New directory" or "New file", but not "New network connection". Or if you're viewing your email inbox, the option you will search under this button is "New email" or maybe "New label", and not "Add new email account". Correct me if I'm wrong, but the network connection is hardly related to the directory list of the internal storage, and this is why many people has difficulty finding this option.

> Yes, for adding new connection, + button makes sense.. populating these things in drawer will spam it. Just think about options such as 'Add SMB connection', 'New SFTP connection' and many more in drawer. However, I do agree that many people face difficulty finding this option.

I think no need for multiple options in the menu. A button like "Add network location" which opens a similar popup to the current solution would be enough.

Opinions?

**Comment by EmmanuelMess** [comment]
Ok, I do agree it makes little sense to have an action in FAB that is not related to context. If there is a nice way to add it to the drawer, I would be for that option.
